src/TodoList.tsx

```diff
@@ -0,0 +1,17 @@
+import { Props, TodoArray } from './types';
+
+const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <>
+      {todos.map((todo: TodoArray) => {
```

Comment:
Typescript should be able to understand how to work with "todo" argument because you already provided type for <Props>

In other words you can remove "TodoArray" type if you want.

Reply:
Done